index.html

```diff
@@ -0,0 +1,61 @@
+            <p class="content__modifier content__matter">Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nulla porro autem quod, officia, velit magni officiis maxime corrupti quam ex accusamus repellendus deleniti ab! Veniam, harum numquam eum laboriosam nisi libero voluptate, assumenda ipsam tempore ea accusantium eveniet deleniti blanditiis! Saepe praesentium blanditiis pariatur. Iure dolorem reprehenderit earum assumenda.</p>
+            <a href="#" class="content__modifier btn btn--pink">Continue Reading</a>
+        </div>
+        <hr>
+        <div class="content__sub">
+                <div class="content__img"></div>
+                <h3 class="content__modifier content__head">Title Two</h3>
+                <p class="content__modifier content__date">Posted on <b>22 June, 2019</b> // Category: Category Two</p>
+                <p class="content__modifier content__matter">Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nulla porro autem quod, officia, velit magni officiis maxime corrupti quam ex accusamus repellendus deleniti ab! Veniam, harum numquam eum laboriosam nisi libero voluptate, assumenda ipsam tempore ea accusantium eveniet deleniti blanditiis! Saepe praesentium blanditiis pariatur. Iure dolorem reprehenderit earum assumenda.</p>
+                <a href="#" class="content__modifier btn btn--pink">Continue Reading</a>
+        </div>
+        <hr>
+        <div class="content__sub">
+                <div class="content__img"></div>
+                <h3 class="content__modifier content__head">Title Three</h3>
+                <p class="content__modifier content__date">Posted on <b>20 June, 2019</b> // Category: Category One</p>
+                <p class="content__modifier content__matter">Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nulla porro autem quod, officia, velit magni officiis maxime corrupti quam ex accusamus repellendus deleniti ab! Veniam, harum numquam eum laboriosam nisi libero voluptate, assumenda ipsam tempore ea accusantium eveniet deleniti blanditiis! Saepe praesentium blanditiis pariatur. Iure dolorem reprehenderit earum assumenda.</p>
+                <a href="#" class="content__modifier btn btn--pink">Continue Reading</a>
+        </div>
+    </div>
+    <div class="form">
+        <label for="sel-category" class="form__select-label"><b>SELECT CATEGORY</b></label>
+        <select id="sel-category" class="form__select-box">
+            <option value="All" selected>All</option>
+            <option value="BBC">BBC</option>
+            <option value="FOX">FOX</option>
+            <option value="NBC">NBC</option>
+        </select>
+
+
+        <label for="subscribe" class="form__subscribe-label"><b>SUBSCRIBE</b></label>
+        <input type="text" id="subscribe" class="form__subscribe-textbox" placeholder="Email Address">
+        <button class="form__subscribe-button">Subscribe</button>
+    </div>
+</div>
+    <footer class="footer">
```

Comment:
alignment issue. To solve this you can use Alt+Shift+F in VSCode

Reply:
Aligned the code using Prettier.